src/graph.cpp

```diff
@@ -210,4 +210,32 @@ void Graph::augmentPath(const int &source, const int &target, const double &valu
             }
         }
     }
+}
+
+
+/**
+* Finds the stations that are at the end of the indicated station's line (i.e have only connection to one other station)
+ * Time Complexity: O(|V|+|E|)
+ * @param station - Id of the starting station
+*/
+std::vector<Vertex *> Graph::findEndOfLines(const int &station) const {
```

Comment:
Why are we referring to a station through an int? Either we refer to it by name, or in this case what this represents is a vertex id and the names/documentation should be updated to be clearer

Reply:
Corrected in new commit